src/app/about/about.component.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <div class="about-card">
+      <h1>About</h1>
+      <div class="about-content">
+        <p>
+          Chef’s Compass is a recipe finding app for you, the user. It helps you find a recipe based on your
+          hunger-tackling desires and any ingredients stocked in your kitchen. Not sure how to use them effectively? Let
+          Chef’s Compass show you the way.
+        </p>
+      </div>
+    </div>
+
+    <div class="cards">
+      <div class="arturo-card">
```

Comment:
li tag for all card display.
Add a same class name since you have the same pattern for displaying card part